core/src/main/scala/spire/math/Interval.scala

```diff
@@ -706,6 +738,31 @@ object Interval {
 
   def apply[A: Order](lower: A, upper: A): Interval[A] = closed(lower, upper)
 
+  /**
+   * Return an Interval[Rational] that corresponds to the error bounds
+   * for the given Double value.
+   * 
+   * The error bounds are represented as a closed interval, whose
```

Comment:
We should probably update the docs to mention the positive/negative infinity case.